### PR TITLE
Adds $not as an Expectation @property to enable IDE autocompletion

### PR DESCRIPTION
**Pull request by eugene-chernogradski**

### CODE REVIEW

File: src/Expectation.php [hunk 1]
```diff
@@ -2,10 +2,11 @@
 namespace Kahlan;
 
 use Exception;
+use Kahlan\Analysis\Debugger;
+use Kahlan\Analysis\Inspector;
 use Kahlan\Code\Code;
 use Kahlan\Code\TimeoutException;
-use Kahlan\Analysis\Inspector;
-use Kahlan\Analysis\Debugger;
+use \Closure;
```
Comment (jails) [inline src/Expectation.php]: No `\` is needed

File: src/Expectation.php [hunk 2]
```diff
@@ -208,11 +211,13 @@ public function __call($matcherName, $params)
...
             $this->_log($result, $data);
+
             return $this;
         }
         $this->_deferred[] = $data + [
-            'instance' => $result, 'not' => $this->_not
-        ];
+                'instance' => $result, 'not' => $this->_not,
+            ];
+
```
Comment (jails) [inline src/Expectation.php]: Looks like it was correctly indented

**Comment by jails** [comment]
Just a couple of typo otherwise good to merge :+1: 
You can mend this commit and force push.

**Comment by eugene-chernogradski** [comment]
All done!

**Pull request merged**